src/Abac.php

```diff
@@ -134,8 +185,8 @@ private function processExtraData(PolicyRuleAttribute $pra, $user, $resource)
                     // The "with" extra data is an array of attributes, which are objects
                     // Once we process it as policy rule attributes, we set it as the main policy rule attribute value
                     $subPolicyRuleAttributes = [];
-                    
-                    foreach ($this->policyRuleManager->processRuleAttributes($data, $user, $resource) as $subPolicyRuleAttribute) {
+                    $subs = $this->policyRuleManager->processRuleAttributes($data, $user, $resource);
```

Comment:
As far as I know, there is no performance issue using a function statement directly inside the foreach. I prefer not to create variables used only one time. Furthermore, the line seems not to be unreadable.

Reply:
issue is in line length. You should to use lines 125 characters by PSR-2 standart.
(also imho to hard read such long call inside foreach)
Suggesting to use ide with PHP_CodeSniffer inside of it to see non-psr lines.

Reply:
True enough